libs/database/src/lib/services/user.service.v2.ts

```diff
@@ -280,4 +284,73 @@ export class UserServiceV2 {
     });
     session.endSession();
   }
+
+  async getBlockedLookup(
+    user: User,
+    { hasRelationshipExpansion, maxResults, sinceId, untilId }: PaginationQuery
+  ) {
+    const filterQuery = {
+      sinceId,
+      untilId,
+      userId: [user._id],
+      blocking: true,
+    };
+
+    const relationships = await this.repositoryService
+      .findRelationships(filterQuery)
+      .sort({ followedUser: SortDirection.DESC })
+      .limit(maxResults)
+      .exec();
+
+    const userIds = relationships.map(({ followedUser }) =>
+      Types.ObjectId(followedUser._id)
+    );
```

Comment:
```suggestion
    const userIds = relationships.map(({ followedUser }) => followedUser);
```

followedUser ที่เก็บไว้ใน relationship เป็น ObjectId อยู่แล้ว